docs/contribute-to-romcal.md

```diff
@@ -125,3 +125,51 @@ _pass the `-t` flag and the path of the file_
 ## Coverage reporting
 
 A coverage report is generated for every core branch build and published to [gh-pages]().
+
+## Committing changes
+
+### Message Format
+
+When you are ready to commit your changes, your commit message should follow the conventional commit format
+
+General format/terminology:
+
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
+```
+
+Here are some guardrails to follow:
+
+When "type" one of the keys, the scopes allowed are as follows:
```

Comment:
The first clause is missing a verb, maybe `is`. 🤔

And what _keys_? l10n keys? 🤔

```suggestion
When the "type" is one of the keys, the scopes allowed are as follows:
```

Reply:
keys are the types in the UL below that indicate which you're checking

Reply:
Sorry, I still don’t follow. What is _UL_?

If _keys_ are _types_, which use two different names of one thing? :thinking:

I really thing I don’t get what you wanted to say by that sentence. Even though I think the problem is with me, just imagine if it is read by someone who don’t understand English that much. And we have such contributors, as we try to localise romcal data (let’s say) to any language and even the calendar data needs to be acquired from different languages. :thinking:

Reply:
the keys of the "map" below the description (`When "type" is one of the keys, the scopes allowed are as follows:`), i.e. `ci`, and the "values" are the descriptions.

how exactly would you say, without being very vague, "these are the allowed types and their allowed scopes"?

Reply:
How about this: _In the following list, you can find what scopes can be used for each allowed type._

Reply:
i think i've got the wording fixed as described. lemme know!

Reply:
Thanks! 🙏